src/Pages/Menu.tsx

```diff
@@ -74,7 +72,15 @@ const Menu: React.FC = () => {
       <div className="profile-screen-background">
         <div className="profile-image-container">
           <div className="profile-image-bubble">
-            <UserOutlined style={{ fontSize: '48px' }} />
+            {uid && imageUrl ? (
+              <img
+                src={new URL(imageUrl).href}
+                alt="avatar"
+                style={{ width: '100%', height: '100%', borderRadius: '50%' }}
```

Comment:
When uploading non-square images, like:
![image](https://github.com/portato-io/portatoapp/assets/24309085/f94fc396-d4b7-4ef2-81e6-0ad8e647280b)

It gets really stretched/squashed:
![image](https://github.com/portato-io/portatoapp/assets/24309085/3b46f1e8-4801-4952-ad33-ea90ef48b705)

Probably something like this would solve this:
https://stackoverflow.com/questions/32477563/how-can-i-fit-images-into-circles-without-stretching

Not sure whether we should just approve this PR, because it's working on a technical level and @emgerold could fix the way it's getting displayed? What do you think, @mAkeddar & @hugo2410 ?

Reply:
Thanks for this one! 
The goal of this PR was to validate the uploading/downloading mechanism, I was hoping to fix the display in a second iteration :) 
What do you think @hugo2410

Reply:
Cool by me. I thought so too; it's kind of a separate issue altogether and probably @emgerold can do this in 1min ^^
Approving :)